### PR TITLE
Added Feature Flag model to the gem

### PR DESCRIPTION
**Pull request by sumitsourav**

### CODE REVIEW

File: app/models/lockstep/feature_flag.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class Lockstep::FeatureFlag < Lockstep::ApiRecord
+  self.model_name_uri = "v1/feature-flags"
+  self.query_path=""
+  load_schema(Schema::FeatureFlag)
+
+  # Create method to override conventional create in lockstep/api_record.rb for sending attrs as an enclosed hash/object
```
Comment (daksh1103) [inline app/models/lockstep/feature_flag.rb:6]: Can you please confirm the below list
1) the lockstep gem test cases are running as expected  without any failures ?
2) Please write testcases for the above changes
3) When accessed from INBOX app the changes work as expected
4) Please check in the inbox side by changing the version of lockstep-rails pointing to new version and check if any testcase is impacted by these changes

File: app/platform_api/schema/feature_flag.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class Schema::FeatureFlag < Lockstep::ApiRecord
```
Comment (rishirajkumar97) [inline app/platform_api/schema/feature_flag.rb:1]: can we add a sample rspec for this model?

File: app/models/lockstep/feature_flag.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  def create
```
Comment (rishirajkumar97) [inline app/models/lockstep/feature_flag.rb:7]: for this method also we can add a Rspec with positive/ negative scenarios

File: app/models/lockstep/feature_flag.rb (hunk 1 above)
Comment (sumitsourav) [reply app/models/lockstep/feature_flag.rb:6]: I have tested the gem in all the scenarios no test cases are failing apart from the known ones.

File: app/platform_api/schema/feature_flag.rb (hunk 2 above)
Comment (sumitsourav) [reply app/platform_api/schema/feature_flag.rb:1]: Added [here](https://github.com/Lockstep-Network/lockstep-sdk-ruby-rails/pull/22/commits/e5385986fc8af5e4ef9876da9898a30f618b2ac2)

File: app/models/lockstep/feature_flag.rb (hunk 3 above)
Comment (sumitsourav) [reply app/models/lockstep/feature_flag.rb:7]: Added [here](https://github.com/Lockstep-Network/lockstep-sdk-ruby-rails/pull/22/commits/e5385986fc8af5e4ef9876da9898a30f618b2ac2)

**Pull request merged**